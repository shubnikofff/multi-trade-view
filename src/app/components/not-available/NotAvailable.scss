@import "../../variables";

.not-available {
  height: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  color: $color-text-black;
  font-weight: bold;

}
