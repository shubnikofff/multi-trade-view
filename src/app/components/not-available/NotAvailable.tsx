import React from 'react';

import './NotAvailable.scss';

interface NotAvailableProps {
    children: React.ReactNode;
}


function NotAvailable({ children }: NotAvailableProps) {
    return (
        <div className="not-available">
            {children}
        </div>
    );
}

export default React.memo<NotAvailableProps>(NotAvailable);
