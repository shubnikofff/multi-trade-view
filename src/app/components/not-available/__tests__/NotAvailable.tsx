import React from 'react';

import { shallow } from 'enzyme';

import NotAvailable from '../NotAvailable';

describe('Component NotAvailable', () => {
    const wrapper = shallow(
        <NotAvailable>
            Some message
        </NotAvailable>
    );

    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
