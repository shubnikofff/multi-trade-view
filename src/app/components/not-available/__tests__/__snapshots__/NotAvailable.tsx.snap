// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component NotAvailable should match snapshot 1`] = `
<div
  className="not-available"
>
  Some message
</div>
`;
